javatests/arcs/core/storage/StorageProxyTest.kt

```diff
@@ -0,0 +1,133 @@
+package arcs.core.storage
+
+import arcs.core.crdt.CrdtData
+import arcs.core.crdt.CrdtModel
+import arcs.core.crdt.CrdtOperation
+import arcs.core.crdt.internal.VersionMap
+import com.google.common.truth.Truth.assertThat
+import com.nhaarman.mockitokotlin2.verifyNoMoreInteractions
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.verify
+import org.mockito.MockitoAnnotations
+
+/** Tests for [StorageProxy]. */
```

Comment:
This isn't strictly needed -- it's pretty clear what class this tests, and IDEs will let you auto-switch

Reply:
Awesome, glad we don't feel the need to exhaustively KDoc very obvious stuff.